app/api/server/lib/email-channel.js

```diff
@@ -0,0 +1,63 @@
+import { EmailChannel as EmailChannelRaw } from '../../../models/server/raw';
+import { EmailChannel, Users } from '../../../models';
```

Comment:
Why are you using raw and regular `EmailChannel` models?
Just the raw model is okay.